src/adapter.js

```diff
@@ -42,7 +42,7 @@ function createQUnitStartFn (tc, runnerPassedIn) { // eslint-disable-line no-unu
           // The fixture is only created once when the runner begins.
           // Resetting is handled by qunit
           var fixture = document.getElementById(FIXTURE_ID)
-          if (!fixture) {
+          if (!fixture && !runner.config.fixture) {
```

Comment:
Could you describe in an inline comment how we would normally get to a situation where the fixture missing, but the fixture was initialised in config? And why we should not to create a fixture in such case?

Reply:
It's in case `QUnit.config.fixture` has [been passed by the user](https://github.com/qunitjs/qunit/blob/89dcd90211fb129df48aa11b72dbafd9f5a18009/runner/fixture.js#L17) (e.g. as a string) and [we should preserve it](https://github.com/qunitjs/qunit/blob/89dcd90211fb129df48aa11b72dbafd9f5a18009/runner/fixture.js#L37). Would you like me to add a comment on that line?

Reply:
Yes, but two points in that case:

1. We should compare to `=== undefined` so that the `null` configuration value (to disable resets) are honoured as well.

2. We still need to create the fixture element, right? Maybe move this check to where we assign `config.fixture`. And also, assuming this always runs after QUnit's own `begin` handler for fixtures, that means we also need to consume the config setting and apply it, right? This is getting rather internal.

Reply:
1. OK
2. QUnit [creates the element for us](https://github.com/qunitjs/qunit/blob/89dcd90211fb129df48aa11b72dbafd9f5a18009/runner/fixture.js#L40).